lillian_klasen/login_reg/templates/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Login and Registration</title>
+</head>
+<body>
+
+    {% with messages = get_flashed_messages()%}
```

Comment:
Try to keep up with indentation in your html files. It's much easier to read if you continue to tab your lines correctly.